releasing/README.md

```diff
@@ -96,50 +78,58 @@ GITHUB_TOKEN=deadbeefdeadbeef
 echo $GITHUB_TOKEN | gh auth login --scopes repo --with-token
 ```
 
+
+Instructions on how to get a new OpenAPI sample can be found in the
+[OpenAPI Readme].
+
 ## Release `kyaml`
 
 #### Establish clean state
 
 ```
-cd ~/gopath/src/sigs.k8s.io/kustomize
 refreshMaster
 testKustomizeRepo
 ```
 
+While you're waiting for the tests, review the commit log. Based on the changes to be included in this release, decide whether a patch, minor or major version bump is needed: [semver review].
+
 kyaml has no intra-repo deps, so if the tests pass,
 it can just be released.
 
-Release it:
+#### Release it
+
+The default increment is a new patch version.
 
 ```
-gorepomod release kyaml --doIt
+gorepomod release kyaml [patch|minor|major] --doIt
 ```
 
 Note the version:
 ```
 versionKyaml=v0.10.20   # EDIT THIS!
 ```
 
-Undraft the release on the [kustomize repo release page],
-make sure the version number is what you expect.
+Undraft the release on the [kustomize repo release page]:
+* Make sure the version number is what you expect.
+* Remove references to commits that aren't relevant to end users of this module (e.g. test commits, refactors).
+* Make sure each commit left in the release notes includes a PR reference.
 
 
 ## Release `cmd/config`
 
-```
-cd ../kustomize
-```
-
-Pin to the most recent kyaml.
+#### Pin to the most recent kyaml
```

Comment:
Should we also add to this section that you have to manually bump two of the transformer kyaml dependencies, perhaps linking to a sample pinning PR?

Reply:
That's what the two new lines of code below do 😄